### ISSUE TITLE
Custom placeholders

### ISSUE THREAD

**Issue by SlimeDog** [filoghost/HolographicDisplays#565]
### Confirmation

- [X] I have read the [documentation](https://filoghost.me/docs/holographic-displays).
- [X] I have read the [FAQ](https://filoghost.me/docs/holographic-displays/faq).

### Description

I have been unsuccessful using custom placeholders. What am I doing wrong?

I would like to display
```
REQUIESCAT
 IN PACE
```

custom-placeholders.yml
```
placeholders:
  '{RIP_top}': ' &fREQUIESCAT '
  '{RIP_bottom}': ' &fIN PACE '
  '{custom_placeholder}': 'This is a custom placeholder.'
  '{primary_color}': '&b'
  '{secondary_color}': '&7'
  '{separator}': '&8&m          '
```

database.yml
```
playground-rip-1:
  lines:
  - '{RIP_top}'
  - '{RIP_bottom}'
  position:
    world: world
    x: 53.5
    y: 66.0
    z: -57.5
```
displays in-game
```
 {RIP_top}
{RIP_bottom}
```

### Holographic Displays version

HolographicDisplays version 3.0.0-SNAPSHOT-b191

**Comment by filoghost** [comment]
There was a bug. Fixed in the latest dev build, thanks for reporting.

**Comment by SlimeDog** [comment]
Confirmed fixed in HolographicDisplays-3.0.0-b193. Thanks.
